code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -136,6 +136,27 @@
 	taste_description = "rich earthy pungent"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/nutriment/clothiment
+	name = "Clothiment"
+	description = "It's not actually a form of nutriment but it does keep Moth people going for a short while..."
+	nutriment_factor = 30 * REAGENTS_METABOLISM
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	brute_heal = 0
+	burn_heal = 0
+	///Amount of satiety that will be drained when the clothiment is fully metabolized
+	var/delayed_satiety_drain = 30
```

Comment:
You should probably make this a define so that it can be reused in nutriment_factor

Reply:
Done